README.md

```diff
@@ -1,4 +1,6 @@
 # [Perun](https://perun.network/) ICP backend
 
 This repository contains the [Internet Computer](https://dfinity.org/) backend for the [go-perun](https://github.com/perun-network/go-perun) channel library.
-This work is financed by the DFINITY grants program, since May 1st 2022, as a follow-up to the initial winter 2021 grant during which we developed a [canister](https://github.com/perun-network/perun-icp-canister) for Perun.
\ No newline at end of file
+This work is financed by the DFINITY grants program, since May 1st 2022, as a follow-up to the initial winter 2021 grant during which we developed a [canister](https://github.com/perun-network/perun-icp-canister) for Perun.
```

Comment:
```suggestion
This project is supported through the DFINITY grants program, since May 1st 2022, as a follow-up to the initial winter 2021 grant during which we developed a [canister](https://github.com/perun-network/perun-icp-canister) for Perun.

## Funding
This project is funded by the DFINITY grants program.
```